### ISSUE TITLE
Address thedarksideof.io as we use a .io TLD

### ISSUE THREAD

**Issue by Mikaela** [privacytools/privacytools.io#1324]
I just asked the team if we can/should do anything to http://www.thedarksideof.io and @blacklight447-ptio suggested me to add a todo.

**Comment by blacklight447-ptio** [comment]
Where would we link this to though?

**Comment by Mikaela** [comment]
I wish we had an about page or section for that. I was also wondering if we could donate to ChagosSupport or ChagosRefugees as PTIO?

**Comment by nitrohorse** [comment]
> I was also wondering if we could donate to ChagosSupport or ChagosRefugees as PTIO?

Hmm, I like this idea!

**Comment by dngray** [comment]
I think we should avoid it. There are many social issues in the world and we don't have a lot of funding to support them all. It is also not really contributing to the message or purpose of privacytools.io

Without endorsement by anyone else really notable it doesn't fill me with confidence either.

I also have some skepticism  regarding the transparency of how the money would be used. They can make claims it would be used for this or that, but how am I expected to believe what is said.

I honestly don't think any amount of money is going to change things considering many large companies use .io domains. You'd have to convince the UK government and ICANN to do something about this issue which is unlikely to happen at this point.

**Comment by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [comment]
To my it looks like a good idea as an outsider to the team, it may not contribute to the message of the website but it makes look at it as trying to be a more ethical organization. Still, I know money isn't abundant so I understand if the support can't be made.

**Comment by dngray** [comment]
It seems they've launched a new website with a bit more information on it, from when I posted my reply above.

**Comment by 5a384507-18ce-417c-bb55-d4dfcc8883fe** [comment]
[Draw.io’s open source diagramming is moving to diagrams.net due to .io ethics & security issues](https://alternativeto.net/news/2020/3/draw-io-s-open-source-diagramming-is-moving-to-diagrams-net-due-to-io-ethics-and-security-issues/)

https://www.diagrams.net/blog/move-diagrams-net

**Comment by FreddyMarsden** [comment]
> TL;DR: I openly urge all FOSS projects and startups to reconsider registering .io ccTLD domains, opting instead for truly generic TLDs like .xyz

[No to .io, yes to .xyz!](https://yarmo.eu/blog/no-io-yes-xyz)

**Comment by LongJohn-Silver** [comment]
> > TL;DR: I openly urge all FOSS projects and startups to reconsider registering .io ccTLD domains, opting instead for truly generic TLDs like .xyz
> 
> [No to .io, yes to .xyz!](https://yarmo.eu/blog/no-io-yes-xyz)

Suzyx is my best friend.

**Comment by jonaharagon** [comment]
At the moment I agree with @dngray, and moving domains is impractical for the organization.
